src/pages/explore/ui/pairs.tsx

```diff
@@ -51,8 +54,49 @@ export const ExplorePairs = () => {
 
   const [search, setSearch] = useState('');
 
+  // Fetch trading pair summaries.
   const { data, isLoading, isRefetching, isFetchingNextPage, fetchNextPage } = useSummaries(search);
```

Comment:
Is it possible that the pairs displayed on the explore page occasionally flip? Compare `SHITMOS / UM` versus `UM / SHITMOS` for instance

------------------------

'<img width="1728" alt="Screenshot 2024-12-17 at 12 28 19 PM" src="https://github.com/user-attachments/assets/315e95b8-cbd4-40fe-8910-d49aca590d62" />

<img width="1728" alt="Screenshot 2024-12-17 at 8 40 06 PM" src="https://github.com/user-attachments/assets/34a4641b-c488-4faf-a684-d645de012f75" />